olmo/optim.py

```diff
@@ -43,6 +43,7 @@ def clip_grads_and_collect_metrics(
         global_step: int,
         collect_param_metrics: bool = True,
         process_group: Optional[dist.ProcessGroup] = None,
+        regularize_embeddings: bool = False,
```

Comment:
Why is this a parameter to this function? Shouldn't it be just captured in the parameter groups? That's how all the other regularization works.